requirements.txt

```diff
@@ -1,9 +1,11 @@
 # setup.py install_requires
 # any changes here should also be reflected in setup.py "install_requires"
-s3fs >= 2021.7.0
-tenacity >= 8.0.1
-ray == 1.10.0
+s3fs >= 2022.11.0
+tenacity >= 8.1.0
+ray[default] >= 2.0.0
```

Comment:
Just curious but any reason for setting *all* dependencies to `>= {version}`? I encountered some slow backtracking time with pip's new dependency resolver (see [here](https://stackoverflow.com/questions/65122957/resolving-new-pip-backtracking-runtime-issue)), which I think was partly due to us not tightening these version requirements.

Reply:
These loose dependency ranges are an artifact of modifying our AWS Glue on Ray tests (uses ```Ray 2.0.0```)  if I recall correctly. In the latest commit, I've now pinned most of the dependencies to what ```pip list``` indicates is actually installed - reducing the ```Install Dependencies``` job runtime from 56 to 35 seconds in one run (https://github.com/pfaraone/deltacat/actions/runs/3836451986/jobs/6530636255).